python/ray/tune/syncer.py

```diff
@@ -93,7 +98,8 @@
     "`RunConfig(storage_path='/mnt/path/to/nfs_storage')`\n"
     "See this Github issue for more details on transitioning to cloud storage/NFS "
     "as well as an explanation on why this functionality is "
-    "being removed: https://github.com/ray-project/ray/issues/37177\n\n"
+    "being removed: https://github.com/ray-project/ray/issues/37177\n"
+    "If you are already using NFS, you can ignore this warning message.\n\n"
```

Comment:
Can this ever happen? If they are already using storage_path, they shouldn't see this message, right?

Reply:
I do log a warning message in all cases when running with multiple nodes, and it'll show up even if NFS is used.

This is because I don't have a good way of checking if `storage_path` is a local directory or NFS.

Is that ok? This warning message will only show up once, and it basically just serves as a soft deprecation reminder.

Reply:
Sounds good to me :)